src/main/java/st/ratpack/auth/AuthModule.java

```diff
@@ -18,13 +17,21 @@ protected void configure() {
 	@Provides
 	@Singleton
 	public TokenValidator tokenValidator(AuthModule.Config config, HttpClient httpClient, ObjectMapper objectMapper) {
-		return new SpringSecCheckTokenValidator(config, httpClient, objectMapper);
+
+		TokenValidator validator = new SpringSecCheckTokenValidator(config, httpClient, objectMapper);
+
+		if (config.cache) {
```

Comment:
We shouldn't be using config to adjust modules like this I've done it other places but its tech debt at this point. We want to module to act a certain way. The module needs renamed to SpringSecAuthModule since that's what it is targeting. 

In the case of caching lets just provide it. users can always overwrite by providing their own validator.